cypress/tests/integration/DataAvailabilityStatements.spec.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file cypress/tests/integration/DataAvailabilityStatements.spec.js
+*/
+
+describe('DataAvailabilityStatements', function () {
+	var statement = 'This is an example of a data availability statement';
+
+	it('Enables Data Availability Statements as submission metadata', function () {
+		cy.login('dbarnes');
+		cy.visit('index.php/publicknowledge/management/settings/workflow#submission/metadata');
+		cy.wait(500);
+        cy.contains('Enable Data Availability Statement metadata').find('[type="checkbox"]').check()
+        cy.wait(500);
+		cy.get('#metadata button').contains('Save').click();
+		cy.get('#metadata [role="status"]').contains('Saved');
+	});
+    
+    it('Adds a statement to a submission', function () {
+        cy.login('dbarnes');
+		cy.visit('index.php/publicknowledge/workflow/index/14/1');
+        cy.get('#publication-button').click();
+        cy.get('#metadata-button').click();
+        cy.get('#metadata-dataAvailability-control-en_US').clear();
+        cy.get('#metadata-dataAvailability-control-en_US').type(statement);
+        cy.get('#metadata button').contains('Save').click();
+        cy.get('#metadata [role="status"]').contains('Saved');
+    });
+
+    it('Creates a review with disclosed author', function () {
+        cy.login('dbarnes');
+		cy.visit('index.php/publicknowledge/workflow/index/14/1');
+		cy.wait(1000);
```

Comment:
`cy.wait` statements [should be considered FIXMEs](https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting). We do have a few cases where we use waits internally -- also needing to be fixed e.g. due to some sort of Javascript rewrite -- but they shouldn't be used indiscriminately or it'll lead to balancing random beakage of the tests against the whole suite taking too long to run.

(For example, there is a 5.5 second delay just below -- that's a lot of time for the server to sit around!)

Reply:
Done, sorry for this bad practice